pkg/cmd/server/api/types.go

```diff
@@ -45,14 +45,17 @@ var (
 	APIGroupBatch       = "batch"
 	APIGroupPolicy      = "policy"
 	APIGroupApps        = "apps"
+	APIGroupFederation  = "federation"
 
 	// Map of group names to allowed REST API versions
 	KubeAPIGroupsToAllowedVersions = map[string][]string{
 		APIGroupKube:        {"v1"},
 		APIGroupExtensions:  {"v1beta1"},
 		APIGroupAutoscaling: {"v1"},
-		APIGroupBatch:       {"v1"},
+		APIGroupBatch:       {"v1", "v2alpha1"},
```

Comment:
I'd prefer not to turn `batch/v2alpha1`, it's not fully functional yet. It'll allow you to create ScheduledJob objects but you won't be able to do anything with it.